lgn-provers/src/provers/v1/preprocessing/euclid_prover.rs

```diff
@@ -221,6 +222,24 @@ impl StorageExtractionProver for EuclidProver {
         )?);
         self.prove(input, "final extraction lengthed")
     }
+
+    fn prove_final_extraction_merge_simple_and_mapping(
```

Comment:
maybe `...merge()`is enough ? no need to specify the two components, right now w only support that ?